internal/cli/opsmanager/clusters/list.go

```diff
@@ -46,11 +46,6 @@ var listTemplate = `ID	NAME	TYPE	REPLICASET NAME{{range .Results}}
 {{.ID}}	{{.ClusterName}}	{{.TypeName}}	{{.ReplicaSetName}}{{end}}
 `
 
-// listAllTemplate used fetching all clusters for all projects.
-var listAllTemplate = `ID	NAME	TYPE{{range .Results}}{{range .Clusters}}
-{{.ClusterID}}	{{.Name}}	{{.Type}}{{end}}{{end}}
```

Comment:
this feature has been broken for a while, we somehow made project id mandatory and this stopped working, I'm removing since trying to add a test for it won't work